app/services/post_scraper.rb

```diff
@@ -143,179 +109,80 @@ def threaded_page_links
 
   def import_post_from_doc(doc)
     subject = @subject || doc.at_css('.entry .entry-title').text.strip
-    logger.info "Importing thread '#{subject}'"
+    Resque.logger.info "Importing thread '#{subject}'"
 
     username = doc.at_css('.entry-poster b').inner_html
-    img_url = doc.at_css('.entry .userpic img').try(:attribute, 'src').try(:value)
-    img_keyword = doc.at_css('.entry .userpic img').try(:attribute, 'title').try(:value)
+    img_node = doc.at_css('.entry .userpic img')
+    img_url = img_node.try(:attribute, 'src').try(:value)
```

Comment:
`img_node.try(:[], 'src')` would do the same thing, afaik – or maybe `img_node&.[]('src')`, but that's a lot uglier